ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.ts

```diff
@@ -708,7 +708,7 @@ export class AddressEmbeddedComponent implements OnDestroy, OnInit {
         if (this.block.requireVerified && !currentAddress) {
             return false;
         }
-        return !(this.block.requirePhone && currentAddress && !(currentAddress.phone));
+        return this.block.requirePhone ? !currentAddress.phone : this.countOfFieldsWithData(currentAddress);
```

Comment:
1. What if currentAddress is null here?

Reply:
3. 
` return this.block.requirePhone ? !currentAddress.phone : ...`
If `this.block.requirePhone===true` and `currentAddress.phone === null`
(means like a block requires a phone number but there is no phone) 
the method `meetsActivityRequirements` returns true. 
Is that behavior correct?

Reply:
these problems are already caught and are being fixed, will update pr soon, thanks